projects/plugins/jetpack/modules/wordads/php/class-wordads-california-privacy.php

```diff
@@ -206,10 +207,10 @@ public static function handle_optout_markup() {
 			<p>%s</p>
 			<p>%s</p>
 			<p>%s</p>',
-			esc_html__( 'If you are a California resident, you have the right to opt out of the "sale" of your "personal information" under the California Consumer Privacy Act ("CCPA")', 'jetpack' ),
-			esc_html__( 'This site operates an ads program in partnership with third-party vendors who help place ads. Advertising cookies enable these ads partners to serve ads, to personalize those ads based on information like visits to this site and other sites on the internet, and to understand how users engage with those ads. Cookies collect certain information as part of the ads program, and we provide the following categories of information to third-party advertising partners: online identifiers and internet or other network or device activity (such as unique identifiers, cookie information, and IP address), and geolocation data (approximate location information from your IP address). This type of sharing with ads partners may be considered a "sale" of personal information under the CCPA.', 'jetpack' ),
+			esc_html__( 'If you are a resident of certain US states, you have the right to opt out of the "sale" of your "personal information" under your state’s privacy laws.', 'jetpack' ),
+			esc_html__( 'This site operates an ads program in partnership with third-party vendors who help place ads. Advertising cookies enable these ads partners to serve ads, to personalize those ads based on information like visits to this site and other sites on the internet, and to understand how users engage with those ads. Cookies collect certain information as part of the ads program, and we provide the following categories of information to third-party advertising partners: online identifiers and internet or other network or device activity (such as unique identifiers, cookie information, and IP address), and geolocation data (approximate location information from your IP address). This type of sharing with ads partners may be considered a "sale" of personal information under your state’s privacy laws.', 'jetpack' ),
```

Comment:
Please escape the apostrophe instead of using a fancy one on these two lines -- "state’s" -- as seen on line 215 below. It should say `state\'s` instead.

Reply:
I'm not sure if there are implications to using the fancy apostrophe. However, searching through the code the fancy `’` is used a lot to represent an apostrophe.

Reply:
We're escaping regular apostrophes elsewhere in this file. That's because fancy semicolons may lead lead to unprintable ￼ artifacts. Please continue follow the pattern in these changes. Consistency is key 🗝️ 

```suggestion
			esc_html__( 'If you are a resident of certain US states, you have the right to opt out of the "sale" of your "personal information" under your state\'s privacy laws.', 'jetpack' ),
			esc_html__( 'This site operates an ads program in partnership with third-party vendors who help place ads. Advertising cookies enable these ads partners to serve ads, to personalize those ads based on information like visits to this site and other sites on the internet, and to understand how users engage with those ads. Cookies collect certain information as part of the ads program, and we provide the following categories of information to third-party advertising partners: online identifiers and internet or other network or device activity (such as unique identifiers, cookie information, and IP address), and geolocation data (approximate location information from your IP address). This type of sharing with ads partners may be considered a "sale" of personal information under your state\'s privacy laws.', 'jetpack' ),
```

Reply:
Interesting. 
There are multiple places it is used in the code https://github.com/Automattic/jetpack/blob/818fb966d1b6737d50e9e0398836db49e54d299d/projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-about-page.php#L155
https://github.com/Automattic/jetpack/blob/0e69b2829d01c0844feea9a1c4a3385956202c1d/projects/plugins/jetpack/class.jetpack-connection-banner.php#L354
https://github.com/Automattic/jetpack/blob/023b5f03f45e474392c37083c26e71ab0ee55914/projects/plugins/jetpack/class-jetpack-recommendations-banner.php#L229

And I believe this translates to the Unicode character [U+2019](https://www.compart.com/en/unicode/U+2019) which seems to be available in the same year as the [single quote U+0027](https://www.compart.com/en/unicode/U+0027) and the fonts we use are able to render this correctly. Except if the fallback fonts on the user's device might miss it but I haven't experienced this or found any conversation about this but I'm actually interested in learning what the caveat here is.

**If I do a string search for `\'` I see results too so I would change to this since it is better to be safe in case this is actually a problem with this.**